src/compare_v.py

```diff
@@ -35,14 +35,15 @@ def import_design(design):
         g.vs[i]["label"] = x.split("/")[-1]
         g.vs[i]["name"] = x
         if cells[x]["IS_PRIMITIVE"] == 1:
-            g.vs[i]["IS_PRIMITIVE"] = 1
+            g.vs[i]["IS_PRIMITIVE"] = True
```

Comment:
Does this need any other changes?  Is `IS_PRIMITIVE` checked anywhere against 1 or 0?